examples/nftvm/actions/outputs.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package actions
```

Comment:
can we rename this file to `errors.go` ?  I think this was copy pasted from a bad example in MorpheusVM (stale code that should have been renamed